packages/adapter/package.json

```diff
@@ -16,5 +18,10 @@
     "@types/node": "18.15.11",
     "eslint": "^8.37.0",
     "typescript": "^5.0.3"
+  },
+  "dependencies": {
+    "@chainsafe/aleo-snap": "workspace:^",
+    "@chainsafe/aleo-snap-types": "workspace:^",
+    "@chainsafe/aleo-snap-wasm": "workspace:^"
```

Comment:
Please remove `@chainsafe/aleo-snap-wasm` as `dependencies` there should not be any usages of wasm in adapter